src/pages/Staking/index.tsx

```diff
@@ -323,13 +335,29 @@ const Staking = (): JSX.Element => {
   const stakedAmount = getStakedAmount();
 
   const availableBalance = React.useMemo(() => {
-    if (!governanceTokenBalance || stakedAmountAndEndBlockIdle || stakedAmountAndEndBlockLoading) return;
+    if (
+      !governanceTokenBalance ||
+      stakedAmountAndEndBlockIdle ||
+      stakedAmountAndEndBlockLoading ||
+      transactionFeeReserveIdle ||
+      transactionFeeReserveLoading ||
+      transactionFeeReserve === undefined
```

Comment:
Just a minor thing. I guess we can just remove `transactionFeeReserve` from this list and add some fail-fast check below like `Something went wrong!` which should not happen.